compair/api/report.py

```diff
@@ -447,6 +453,102 @@ def peer_feedback_report(course, assignments, group_name):
 
     return report
 
+def comparison_result_report(title, course, assignments, group_name):
+    report = []
+
+    classlist = UserCourse.query \
```

Comment:
Safer to state explicitly what we want to retrieve?  e.g. if new roles added in the future, it is easy to see if a report / function not covering the new roles.

Reply:
In this cause I think its making it more complex since you're retrieving them separately and then combining them in two places (and that's all the two separate lists are used for). I don't see why you'd want to run two queries